### DISCUSSION TITLE
Easy Translator Tool became slow - ideas?

### DISCUSSION THREAD

**Discussion by Robba91** [MscrmTools/XrmToolBox#1144] [Q&A] [answered]
Dear Guys!

I absouletly love to work with the XRMToolbox - nice tool!

But I have noticed a huge performance drop when it comes to the usage of the Easy Translator tool.
I've noticed that already within the last year.

So for me it is not possible to add more batches as 10 -> otherwise the system will run into a time out...

I'm not sure if this is caused by our internal network changes (we did some firewall changes last year) or something similar or if anyone else noticed the same issues?
Any ideas how to get the old performance back?

Thanks a lot!

BR
Robin

**Comment by MscrmTools** [comment, marked as answer]
I guess it's just because operations on metadata are really time consuming and there is nothing we can do here.

Only suggestion is to raise your timeout configuration for connection above the standard 2 minutes so that operations can last longer.

**Reply by Robba91** [reply]
Thanks! Will try that out!

**Reply by lostincoding** [reply]
> Only suggestion is to raise your timeout configuration for connection above the standard 2 minutes so that operations can last longer.

@MscrmTools How do I increase the time out? I looked around in XrmToolbox configuration and I have no idea where such setting is.

**Reply by lostincoding** [reply]
> Thanks! Will try that out!

@Robba91 What was the solution that worked for you ? Can you update this thread so others benefit from it? (me included)
I think I'm in the same boat you once were on.. Importing taking too much time and eventually failing with a timeout error.

**Reply by lostincoding** [reply]
Well, decreasing the batch count and trying again was enough.
It started slow and it seemed like it was going to take forever, but in total it was just 30 minutes.

**Reply by MscrmTools** [reply]
The timeout is set at the connection level. Edit your connection in Connection Manager to put a higher value

**Reply by Robba91** [reply]
@lostincoding I used the method which was already described by @MscrmTools  👍 

I changed the timeout from 2 Minutes to 30 Minutes, that fixed my problem.
But I'll also consider to decrease the batch load, because it still feels a bit slower as it was in the past.

**Reply by lostincoding** [reply]
@Robba91 @MscrmTools I dont see where that setting is...
I even updated to the latest XrmToolbox version (I realized I had an older version).
![000267 -Window](https://user-images.githubusercontent.com/48190080/191925616-3443afa1-dc00-4397-9f4d-fa8e824344a1.png)
